corehq/apps/hqwebapp/views.py

```diff
@@ -351,6 +351,11 @@ def login(req):
     # this view, and the one below, is overridden because
     # we need to set the base template to use somewhere
     # somewhere that the login page can access it.
+
+    if settings.SERVER_ENVIRONMENT == 'icds':
```

Comment:
But future clusters won't redirect specifically to `icds-cas`, so I'd also need some kind of server_environment/domain mapping, we'll be updating this code in some way for future clusters anyway.